assets/src/edit-story/components/panels/design/videoOptions/videoOptions.js

```diff
@@ -52,14 +64,62 @@ const StyledCheckbox = styled(Checkbox)`
 `;
 
 function VideoOptionsPanel({ selectedElements, pushUpdate }) {
+  const isMuteVideoEnabled = useFeature('enableMuteVideo');
+  const { isTranscodingEnabled } = useFFmpeg();
+  const { muteExistingVideo } = useLocalMedia((state) => ({
+    muteExistingVideo: state.actions.muteExistingVideo,
+  }));
+  const resource = getCommonValue(selectedElements, 'resource');
+  const { isMuted, isTranscoding, isMuting, local } = resource;
   const loop = getCommonValue(selectedElements, 'loop');
+  const isSingleElement = selectedElements.length === 1;
+
+  const handleHandleMuteVideo = useCallback(() => {
+    muteExistingVideo({ resource });
+  }, [resource, muteExistingVideo]);
+
+  const canMuted = useMemo(() => {
+    return (
+      (isMuteVideoEnabled &&
+        isTranscodingEnabled &&
+        !local &&
+        !isMuted &&
+        !isTranscoding &&
+        isSingleElement) ||
+      isMuting
+    );
+  }, [
+    isMuteVideoEnabled,
+    isTranscodingEnabled,
+    local,
+    isMuted,
+    isTranscoding,
+    isSingleElement,
+    isMuting,
+  ]);
 
   const checkboxId = `cb-${uuidv4()}`;
+  const buttonText = isMuting
+    ? __('Muting video', 'web-stories')
+    : __('Mute video', 'web-stories');
   return (
     <SimplePanel
       name="videoOptions"
       title={__('Video Settings', 'web-stories')}
     >
+      {canMuted && (
+        <Row>
+          <StyledButton
+            disabled={isMuting}
+            variant={BUTTON_VARIANTS.RECTANGLE}
+            type={BUTTON_TYPES.SECONDARY}
+            size={BUTTON_SIZES.SMALL}
+            onClick={handleHandleMuteVideo}
+          >
+            {buttonText}
+          </StyledButton>
+        </Row>
+      )}
```

Comment:
In my testing the button text never changed.

1. I click on button
2. Button is disabled, muting starts
3. Muting finishes, button gets removed

Reply:
Are you not seeing [this](https://user-images.githubusercontent.com/237508/126512610-794b012d-7614-45fa-883b-fad3bdec859d.mov)?

Reply:
I am now 👍